utils/build-script

```diff
@@ -395,6 +395,18 @@ class BuildScriptInvocation(object):
             args.stdlib_deployment_targets.append(
                 StdlibDeploymentTarget.Android.armv7.name)
 
+        # Infer platform flags from manually-specified configure targets.
+        # This doesn't apply to Darwin platforms, as they are
+        # already configured. No building without the platform flag, though.
+
+        android_tgts = [tgt for tgt in args.stdlib_deployment_targets
+                        if StdlibDeploymentTarget.Android.contains(tgt)]
+        if not args.android and len(android_tgts) > 0:
+            args.android = True
+            args.skip_build_android = True
+
+# ---
```

Comment:
This is there to fix a PEP error about too many blank lines. I assumed 2 lines were left blank to separate the initialiser from the massive block of parsing code above.